frontend/src/pages/UpdateUser.tsx

```diff
@@ -1,5 +1,139 @@
+    setUserDetails({
+      ...userDetails,
+      [name]: value,
+    });
+  };
+
+  const handleSubmit = async () => {
+    console.log(userDetails);
+    // TODO:
+    // Hit the correct API to successfully store info to the db
+    const res = await fetch('http://localhost:3001/updateUser', {
+      method: 'POST',
+    });
+    await res.json();
+  };
+
+  return (
+    <Background bgcolor={'#fff'}>
+      <Grid
+        item
+        xs={12}
+        sx={{
+          display: 'flex',
+          flexDirection: 'column',
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}
+      >
+        <Typography variant={'h5'}>This is...</Typography>
+        <Typography variant={'h2'}>TuesHey</Typography>
+        <Typography variant={'h4'}>
+          Let's get to know you <br /> ✌️😙✌️
+        </Typography>
+      </Grid>
+      <Grid
+        item
+        xs={12}
+        sx={{
+          display: 'flex',
+          flexDirection: 'column',
+          justifyContent: 'space-around',
+          alignItems: 'center',
+        }}
+      >
+        <div style={{ width: '25rem' }}>
+          <TextField
+            onChange={handleChange}
+            value={userDetails.emojis}
+            placeholder={'😎 ⚽ 🍔'}
+            name="emojis"
+            label={'Describe yourself in 3 emojis!'}
+            fullWidth
+          />
+        </div>
+        <div style={{ width: '25rem' }}>
+          <FormControl fullWidth>
+            <InputLabel>Choose your faculty</InputLabel>
+            <Select
+              value={userDetails.faculty}
+              label="Choose your faculty"
+              onChange={handleChange}
+              name="faculty"
+            >
+              <MenuItem value={Faculty.ArtsDesignAndArchitecture}>
+                Arts, Design & Architecture
+              </MenuItem>
+              <MenuItem value={Faculty.Business}>Business School</MenuItem>
+              <MenuItem value={Faculty.Engineering}>Engineering</MenuItem>
+              <MenuItem value={Faculty.LawAndJustice}>Law and Justice</MenuItem>
+              <MenuItem value={Faculty.MedicineAndHealth}>
+                Medicine and Health
+              </MenuItem>
+              <MenuItem value={Faculty.Science}>Science</MenuItem>
+            </Select>
+          </FormControl>
+        </div>
+        <div style={{ width: '25rem' }}>
+          <TextField
+            label={"What's your favourite place to eat on campus?"}
+            value={userDetails.faveEat}
+            onChange={handleChange}
+            name="bestEat"
+            fullWidth
+          />
+        </div>
+        <div style={{ width: '25rem' }}>
+          <TextField
+            label={"I'm into..."}
+            value={userDetails.interests}
+            placeholder={'hip-hop, Simpsons'}
+            onChange={handleChange}
+            name="interests"
+            fullWidth
+          />
+        </div>
+      </Grid>
```

Comment:
There's no spacing between the textfields which looks a little weird. Rather than using divs, use an item Grid. You can set the parent grid (in line 61) to being a container (note: grids can be container AND item so just add `container` without removing the `item`). After you do this, you can add gaps between the textfields. Take a look at the hint I put in [this](https://github.com/Siyer2/uni-connector/pull/26/files#r1045323424) comment